sdk/ai/azure-ai-projects/CHANGELOG.md

```diff
@@ -15,7 +15,7 @@ Please see new samples and package README.md file.
 
 * Azure AI Foundry Project endpoint is now required to construct the `AIProjectClient`. It has the form
 `https://<your-ai-services-account-name>.services.ai.azure.com/api/projects/<your-project-name>`. Find it in your AI Foundry Project
-Overview page. The factory method `from_connection_string` was removed.
+Overview page. The factory method `from_connection_string` was removed. The support of connection strings and hub based projects was dropped. Please create a new Azure AI Foundry resource, which uses endpoints (recommended), or pin the version of `azure-ai-projects` to `1.0.0b10` if it is not possible.
```

Comment:
The same suggestion

Support for project connection string and hub-based projects has been discontinued. We recommend creating a new Azure AI Foundry resource utilizing project endpoint. If this is not possible, please pin the version of Azure.AI.Projects to version 1.0.0-beta.8 or earlier.